ocaml/lib/snarky_bindings.mli

```diff
@@ -51,6 +51,23 @@ val snarky :
           Js.meth >
       Js.t
       Js.readonly_prop
+  ; gates :
+      < rangeCheck0 :
+          (   Field.t
```

Comment:
I would be in favor of documenting a bit the parameters as we are in an interface file :)